### PR TITLE
Add nbserverproxy doc

### LINKED ISSUE #111: Include nbserverproxy or describe how to use it in the docs
See https://github.com/dask/dask-jobqueue/issues/80#issuecomment-400797727.

To prevent from doing a lot of ssh tunneling, using [`nbserverproxy`]( https://github.com/jupyterhub/nbserverproxy ) could be helpful!

### PR DESCRIPTION
**Pull request by guillaumeeb**
Closes #111.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -89,6 +89,9 @@ Once Jupyter is installed and configured, using a Jupyter notebook is done by:
 
    $ ssh -N -L 8888:x.x.x.x:8888 username@hpc_domain
 
+Now you can fo to ``http://localhost:8888`` on your browser to access the
```
Comment (mrocklin) [inline docs/index.rst]: Typo: `fo to`

**Comment by mrocklin** [comment]
This looks good to me.  You might also want to add a note that they can change the `distributed.dashboard.link` config entry to change the links shown from the the Cluster and Client objects.

**Comment by guillaumeeb** [comment]
> You might also want to add a note that they can change the distributed.dashboard.link config

Problem is: I did not find a value that was redirecting correctly to the proxy page. Maybe because once my notebook is started, I navigate into folders, and thus I'm not in notebook root folder anymore.

See my comment here https://github.com/pangeo-data/pangeo/pull/304#discussion_r208144121.

**Comment by guillaumeeb** [comment]
I finally find the solution thanks to dask-examples repo. The link must obviously be set differently using either Jupyterhub or manually launched notebook.

**Pull request merged**